blocks/init/src/Blocks/components/layout/components/layout-options.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { __, sprintf } from '@wordpress/i18n';
+import { RangeControl } from '@wordpress/components';
+import {
+	icons,
+	IconLabel,
+	getAttrKey,
+	checkAttr,
+	getOption,
+	SimpleVerticalSingleSelect,
+	CollapsableComponentUseToggle
+} from '@eightshift/frontend-libs/scripts';
+import manifest from '../manifest.json';
+
+export const LayoutOptions = (attributes) => {
+	const {
+		title: manifestTitle,
+		attributes: reset,
+		combinations,
+	} = manifest;
+
+	const {
+		setAttributes,
+		label = manifestTitle,
+		layoutShowControls = true,
+
+		showLayoutUse = true,
+		showLayoutType = true,
+		showLayoutTotalItems = true,
+	} = attributes;
+
+	if (!layoutShowControls) {
+		return null;
+	}
+
+	const layoutUse = checkAttr('layoutUse', attributes, manifest);
+	const layoutType = checkAttr('layoutType', attributes, manifest);
+	const layoutTotalItems = checkAttr('layoutTotalItems', attributes, manifest);
```

Comment:
This naming is a bit odd, the label says `Maximum items to show`, would it be better to name it `layoutMaxNumberOfItems`?

Reply:
i like the keep the attribute naming as short and simple as possible. The label is here for the users. So i think this one is ok